plugins/ssl/multi_ssl

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
This plugin uses only a single bash-specific expression (`SITE="${1/_*/}"`). Thus maybe you want to avoid the dependency on bash and stick with `sh` instead?